main/zoe/api/zoe.md

```diff
@@ -108,6 +131,10 @@ const invitationValue = await E(zoe).getInvitationDetails(invitation);
 Takes bundled source code for a Zoe contract as an argument and installs the code on Zoe.
 Returns an `installation` object. 
 
+An `installation` is an object with two properties:
+- `installationHandle`: An opaque identifier, used as the table key **tyg todo: What table?**
```

Comment:
The installationHandle is gone. installation is now an object. Its only method is getBundle(), which returns the source for the contract as a string.